cmd/strobe/README.md

```diff
@@ -0,0 +1,27 @@
+# Strobe
+
+Strobe is a controller that implements the periodic jobs defined in the
+Lighthouse config ConfigMap:
+
+```yaml
```

Comment:
I am not sure if it should be defined in a configmap. Normally in jx, configmap would be defined/configured in the cluster git repo, which should be normally off limits for repository owners. It should be in the same repo, so if a repository author wants to define a periodics, they should be able to define a new periodics job in the triggers yaml, wdyt?
May be a block like pre-submit/post-submit?

Reply:
Also sorry to reference other repositories, but would it be possible to get a review for this PR please? https://github.com/jenkins-x/go-scm/pull/321
We depend on the poller component being able to see if a particular job has ever succeeded (rather than just the latest run) which is the reason for the change. We are using a custom fork with that PR at the moment

Reply:
Thanks for the approval! I have created another PR to bump the go-scm version used by Lighthouse: https://github.com/jenkins-x/lighthouse/pull/1493

Reply:
I suppose the periodics would be registered similar to how pipelines are cached (but periodics will store instead). It being registered within the cluster repo goes away from the jenkins-x ideaollogy of being repo and developer centric as it would make you have to jump through 2 repos to add a feature to your repo